canonical/snapcraft.io
# Update headings in brand store

## Done
Updated the headings and breadcrumbs in the brand store

## How to QA
- Go to https://snapcraft-io-4548.demos.haus/admin
- Check that the "Store snaps", "Members", and "Settings" views have the store name and section title
- Go to https://snapcraft-io-4548.demos.haus/admin/njwQYXFnS7ppo7LaGxoh7aqVZc1CPi26/models
- Click through to a model
- Check that the breadcrumb at the top is styled as a heading like the "Store snaps/Members/Settings" views

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-8961
